circuits/ballotproof/wasm_inputs.go

```diff
@@ -0,0 +1,108 @@
+package ballotproof
+
+import (
+	"fmt"
+	"math/big"
+
+	"github.com/iden3/go-iden3-crypto/mimc7"
+	"github.com/vocdoni/vocdoni-z-sandbox/circuits"
+	"github.com/vocdoni/vocdoni-z-sandbox/crypto"
+	bjj "github.com/vocdoni/vocdoni-z-sandbox/crypto/ecc/bjj_gnark"
+	"github.com/vocdoni/vocdoni-z-sandbox/crypto/ecc/format"
+	"github.com/vocdoni/vocdoni-z-sandbox/crypto/elgamal"
+	"github.com/vocdoni/vocdoni-z-sandbox/types"
+)
+
+// WasmVoteInputs composes the data to generate the inputs required to generate
+// the witness for a ballot proof using the circom circuit and also the data
+// required to cast a vote sending it to the sequencer API. It receives the
+// BallotProofWasmInputs struct and returns the BallotProofWasmResult struct.
+// This method parses the public encryption key for the desired process and
+// encrypts the ballot fields with the secret K provided. It also generates
+// the commitment and nullifier for the vote, using the address, process ID
+// and the secret provided.
+func WasmVoteInputs(
```

Comment:
Why name it `wasm`? I think we can have this function as a helper to format the inputs, used by the WASM, but also other components (i.e a potential Go client or the end2end test). Maybe `FormatCircomBallotInputs()`